lib/types/entry.ts

```diff
@@ -339,7 +339,7 @@ export type EntryCollection<
 > = ContentfulCollection<Entry<EntrySkeleton, Modifiers, Locales>> & {
   errors?: Array<any>
   includes?: {
-    Entry?: any[]
-    Asset?: any[]
+    Entry?: Entry<EntrySkeletonType, Modifiers, Locales>[]
```

Comment:
I think we should not pass down the `Locales` variable here and use `LocaleCode` instead because linked entries might come from other spaces which can have different locales.

Reply:
So when we make cross-space requests we basically don't know anymore what locales we're dealing with?
That makes sense, but won't that then also already be broken in the more common case of actually using the link resolution, since there we cannot distinguish?

I'm a bit split between making it easy for the most common use case of not having cross space links, but a set of explicit locales, and ensuring that the types aren't misleading in the case where you use cross space links to spaces with a different set of locales.

I guess since the includes are also not returned in a structure where customers could provide something like a space / locale mapping that means we basically have no way to ever really have locale types that are anything other than `string`.

So I'm wondering, maybe it's better to keep using `Locales` which will probably in much more than 90% of cases be the expected behavior, and which you can at least put all the locales between the different spaces in as a workaround, instead of forcing `includes` to always have untyped locales.

Reply:
Yeah, you’re right. Even in most cross space cases the locales should be created to match and in cases that are more complicated the locale types probably won’t help much and have to be checked at runtime anyway in which case you would just use `string` and be correct.